### DISCUSSION TITLE
Face detection using AI deck

### DISCUSSION THREAD

**Discussion by Bikerholic** [bitcraze/discussions#102] [Q&A]
Hi 
 I am trying to take out video stream from AI deck please tell me how to do it and how to implement face detection.

**Comment by knmcguire** [comment]
There is currently an issue on the current version of the face detection example. See this issue: https://github.com/bitcraze/aideck-gap8-examples/issues/92

There is no immediate plan yet but roughly planning to see if we can convert t[his example of greenwaves to include cpx](https://github.com/GreenWaves-Technologies/face_detection). But we have to find the time for this still!

If you want to have a go at it allready, go for it! You can use the wifi example of the aideck-gap8-examples repo as a guideline to setup streaming.
